### DISCUSSION TITLE
Relative expression out

### DISCUSSION THREAD

**Discussion by user1maria** [Open-Systems-Pharmacology/Forum#1245] [Questions & Problems]
Hello, 

I am including TMDD in my model in Mobi and a question came up about protein expression:

What is the difference between „Relative expression out” and „Relative expression normalized”?
Why can I change the Formula to calculate „Relative expression out” in “Parameter start values” and what does it mean if I change it e.g. to “RelExpOutFromNorm”?

Thank you :)

**Comment by StephanSchaller** [comment]
Normalized expressions are the relative expressions normalized to the maximum relative expression.
MoBi allows changing anything if you want. This does not mean you should. 
Do not change the formula assignments. What happens is that a wrong formula will be used for the parameter.

Hope this helps!
Stephan

**Reply by user1maria** [reply]
Thank you Stephan for your answer!
I understand that normalized expressions are the relative expressions normalized to the maximum relative expression. But what do I need "Relative expression out" for?

Thanks :) Maria

**Reply by Yuri05** [reply]
"Relative expression out" was the "old" way (in PK-Sim 9 and before) to take different protein localization settings into account.
Actually the name "Relative expression out" is even misleading. Normally you should not touch those parameters.

https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile#localizations-and-initial-concentrations-of-enzymes
If you check in the current documentation (the link above) how the initial concentrations of proteins are calculated - the highlighted parts of the formulas correspond to what was previously called "Relative expression out"
![grafik](https://user-images.githubusercontent.com/25061876/182600161-db53b172-f5ca-40ac-8e2c-996421fd1738.png)

If possible, I would suggest that you take the basis PBPK model from PK-Sim 10 or 11 and extend it in MoBi with TMDD - would make the life easier i the future (hopefully :))
